cmd/admin-heal-ops.go

```diff
@@ -543,24 +541,64 @@ func (h *healSequence) healConfig() error {
 		return errServerNotInitialized
 	}
 
-	configFile := path.Join(minioConfigPrefix, minioConfigFile)
-	configBackupFile := path.Join(minioConfigPrefix, minioConfigBackupFile)
-	for _, cfg := range []string{configFile, configBackupFile} {
-		res, err := objectAPI.HealObject(h.ctx, minioMetaBucket, cfg, h.settings.DryRun)
+	// NOTE: Healing on configs is run regardless
+	// of any bucket being selected, this is to ensure that
+	// configs are always uptodate and correct.
+	marker := ""
+	isTruncated := true
+	for isTruncated {
+		if globalHTTPServer != nil {
```

Comment:
I am pretty sure that config files under .minio.sys/config are small and it seems we really don't need reporting healing here, what do you think @harshavardhana ?

Besides, let's have a generic way to deal with report healing if we want in another PR

Reply:
For now, I am keeping it this way to move ahead with the functionality, we can think about it another round of changes. Especially we also report about policy.json etc which also doesn't need to be communicated.